docs/quickstart.rst

```diff
@@ -107,5 +107,124 @@ to describe more complex demographic histories.
     ts = msprime.sim_ancestry(samples=samples, demography=demography)
     ts.tables.nodes
 
-.. todo:: This example is rough and needs to be updated once the demography
-    API is sharpened up a bit.
+.. todo:: Links into more detailed documentation
+
+******************
+Upgrading from 0.x
+******************
+
+This section is to help 0.x users of the get up to speed quickly, summarising the new
+APIs and their main differences to what you are used to.
+
+The main change is that there are two new functions, :func:`.sim_ancestry` and
+:func:`.sim_mutations` which correspond to the 0.x functions :func:`.simulate`
+and :func:`.mutate`. The 0.x functions are **deprecated** but **will continue
+to be supported indefinitely**.
+
++++++++++++++++++++++++
+Backwards compatibility
++++++++++++++++++++++++
+
+All existing simulations should work as before, *except* for simulations relying on
+the detailed properties of RecombinationMaps. If your code uses the ``num_loci``
+property, then it may need to be updated. The reason for this is that ``msprime``
+has changed to simulate directly in physical coordinates internally (which has
+greatly simplified the code and solved many thorny issues) and this is fundamentally
+incompatible with the approach taken in 0.x. In the vast majority of cases, this
+will have no effect.
+
+If you are using ``num_loci`` to simulate a discrete genome, it may be simplest to
+convert your code to use the new :func:`.sim_ancestry` method. If you were following
+a recipe to simulate multiple chromosomes under the DTWF model, please see
+the :ref:`updated recipe <sec_ancestry_multiple_chromosomes>`.
+
+++++++++
+Ancestry
+++++++++
+
+The new :func:`.sim_ancestry` function replaces the 0.x :func:`.simulate`
+function and is very similar. There are some important differences though:
+
+* Samples are now **individuals** rather **nodes** (i.e. monoploid
```

Comment:
This is a bit confusing - in the *tree sequence*, "being a sample" is still a property of a node, not an individual! How about
```
* The `samples` argument now refers to the **number of individuals** rather than **the number of nodes**
```

Reply:
Done.